### ISSUE TITLE
Unable to update YouTube Music

### ISSUE THREAD

**Issue by EmVeddii** [YTVanced/VancedManager#452]
**Phone Specifications:**
- Brand: Samsung Galaxy
- Operating System: Android; One UI
- Android Version: A 11; 1UI 3.1
- Vanced Manager Version: 2.5.1 (Weed)

I got a notification stating I could install the new version of YouTube Music. It either said the latest was unavailable, or it let me and then was unable to download.

**Steps to reproduce:**

- Open Vanced Manager
- View YouTube Vanced
     - If it says "Unavailable" on the "Latest"
       Version, that's an issue.
     - If it allows you to Update, click it, and 
       it will say "Error Downloading 
       nonroot.apk" and won't install.

**Further details:**

Basically, it says I can download the new Version of YouTube Music. I am on Vanced Manager 2.5.1 on a Samsung Galaxy S20+, Android 11, One UI 3.1. When I opened the app the first time (and a few times since) it said "Unavailable" next to "Latest:", and it would only let me Reinstall. After restarting (and a few times since) it said "4.18.50" next to "Latest:" and would let me Update, however when I clicked Update the bar didn't move and it said "Error Updating nonroot.apk" underneath.

**Comment by Hachimann** [comment]
I have the same problem

**Comment by X1nto** [comment]
duplicate of #450
